packages/cli/cli/src/commands/validate/validateDocsBrokenLinks.ts

```diff
@@ -0,0 +1,37 @@
+import { validateDocsWorkspace } from "@fern-api/docs-validator";
+import { OSSWorkspace } from "@fern-api/lazy-fern-workspace";
+import { Project } from "@fern-api/project-loader";
+
+import { CliContext } from "../../cli-context/CliContext";
+import { logViolations } from "./logViolations";
+
+export async function validateDocsBrokenLinks({
+    project,
+    cliContext,
+    errorOnBrokenLinks
+}: {
+    project: Project;
+    cliContext: CliContext;
+    errorOnBrokenLinks: boolean;
+}): Promise<void> {
+    const docsWorkspace = project.docsWorkspaces;
+
+    if (docsWorkspace == null) {
+        // Should this fail and throw?
```

Comment:
Probably, unless `validateDocsBrokenLink` is called on _every_ Project instance, regardless of whether or not it has a docs configuration.

A no-op like it is now is fine in that case.